package.json

```diff
@@ -5,6 +5,8 @@
     "glob": "^7.2.0",
     "license-check-and-add": "4.0.5",
     "nyc": "17.1.0",
+    "package-json-from-dist": "^1.0.0",
+    "path-scurry": "^2.0.0",
```

Comment:
What are these packages?

Reply:
Dependencies of `rimraf` and `glob`. 

Removed.